src/utils/removeTodo.tsx

```diff
@@ -0,0 +1,29 @@
+import { deleteTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+
+type Args = {
+  deletedId: number;
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setLoadingItemsIds: React.Dispatch<React.SetStateAction<number[]>>;
+  handleError: (errMessage: string) => void;
+};
+
+export const RemoveTodo = ({
```

Comment:
Why did you capitalize the name of this function?

Let's move it inside the context